package.json

```diff
@@ -110,8 +110,6 @@
     "jszip": "^3.5.0",
     "mkdirp": "^0.5.1",
     "mocha": "8",
-    "mocha-junit-reporter": "^2.0.0",
```

Comment:
we don't want junit reports anymore? I think it's fine to remove them here. I can add them back later on if we think they can be useful

Reply:
CircleCI is able to use them in the UI, but GitHub Actions doesn't have such functionality. If this is something you're using locally I can keep bring it back.

Reply:
feel free to leave it out. I can add it back later to dogfood ci visibility